data.sql

```diff
@@ -13,4 +13,64 @@ VALUES ('Charmander', '2020-02-08', 0, false, -11 ),
 ('Angemon', '2005-06-12', 1, true, -45 ),
 ('Boarmon', '2005-06-07', 7, true, 20.4 ),
 ('Blossom', '1998-10-13', 3, true, 17 ),
-('Ditto', '2022-05-14', 4, true, 22 );
\ No newline at end of file
+('Ditto', '2022-05-14', 4, true, 22 );
+
+INSERT INTO owners(full_name,age )
+VALUES('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species(name )
+VALUES('Pokemon'),
+('Digimon');
+
+UPDATE animals
+SET species_id = 2
+WHERE name ILIKE '%mon%';
+
+UPDATE animals
+SET species_id = 1
+WHERE NOT name ILIKE '%mon%';
+
+BEGIN;
+UPDATE animals
+SET owner_id = 1
+WHERE name = 'Agumon';
+
+UPDATE animals
+SET owner_id = 2
+WHERE name = 'Gabumon' OR name = 'Pikachu';
+
+UPDATE animals
+SET owner_id = 3
+WHERE name = 'Devimon' OR name = 'Plantmon';
+
+UPDATE animals
+SET owner_id = 4
+WHERE name = 'Charmander' OR name = 'Squirtle'  OR name = 'Blossom';
+
+UPDATE animals
+SET owner_id = 5
+WHERE name = 'Angemon' OR name = 'Boarmon';
+
+COMMIT;
+
+
+SELECT name FROM animals JOIN owners ON animals.owner_id = owners.id 
+WHERE full_name = 'Melody Pond';
+
+SELECT name FROM animals JOIN species ON animals.species_id = species.id 
```

Comment:
* ## I got an error trying to run this query, Kindly fix it 
* ## Also kindly note that all queries should be moved to your `queries.sql` file 
![image](https://user-images.githubusercontent.com/52497246/236370699-e582b045-53d1-4be1-9406-aa94cf17c376.png)

![image](https://user-images.githubusercontent.com/52497246/236370492-581d2bf0-b190-4737-a60a-eb55f061a00f.png)